admin_app/src/app/dashboard/components/TabPanel.tsx

```diff
@@ -26,3 +37,4 @@ const TabPanel: React.FC<TabPanelProps> = ({ tabValue, handleChange }) => {
 };
 
 export default TabPanel;
+export type { Period };
```

Comment:
What's the use for this ?

Reply:
I should be importing these from types as you suggested above. I'll fix this.